Description: Adds /usr/share/openstack-dashboard in the search path for manage.py
Author: Mehdi Abaakouk <sileht@sileht.net>
Forwarded: not-needed

Index: horizon/manage.py
===================================================================
--- horizon.orig/manage.py	2013-12-09 22:14:14.000000000 +0800
+++ horizon/manage.py	2013-12-10 00:30:58.000000000 +0800
@@ -17,6 +17,8 @@
 
 from django.core.management import execute_from_command_line  # noqa
 
+sys.path.append("/usr/share/openstack-dashboard")
+
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE",
                           "openstack_dashboard.settings")
